python312Packages.rns: 0.8.6 -> 0.8.7, python312Packages.nomadnet: 0.5.4 -> 0.5.5

fabaff opened the pull request:
<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

fabaff committed:
python312Packages.rns: 0.8.6 -> 0.8.7

Diff: https://github.com/markqvist/Reticulum/compare/refs/tags/0.8.6...0.8.7

Changelog: https://github.com/markqvist/Reticulum/releases/tag/0.8.7

fabaff committed:
python312Packages.nomadnet: 0.5.4 -> 0.5.5

Diff: https://github.com/markqvist/NomadNet/compare/refs/tags/0.5.4...0.5.5

Changelog: https://github.com/markqvist/NomadNet/releases/tag/0.5.5

fabaff commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 363954`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 12 packages built:</summary>
  <ul>
    <li>python311Packages.lxmf</li>
    <li>python311Packages.lxmf.dist</li>
    <li>python311Packages.nomadnet</li>
    <li>python311Packages.nomadnet.dist</li>
    <li>python311Packages.rns</li>
    <li>python311Packages.rns.dist</li>
    <li>python312Packages.lxmf</li>
    <li>python312Packages.lxmf.dist</li>
    <li>python312Packages.nomadnet</li>
    <li>python312Packages.nomadnet.dist</li>
    <li>python312Packages.rns</li>
    <li>python312Packages.rns.dist</li>
  </ul>
</details>

qbit commented:
hah! https://github.com/NixOS/nixpkgs/pull/363947 :D 

I was gonna wait on nomadnet because I wanted to add myself as maintainer. Yours works for me though.

qbit commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 363954`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 12 packages built:</summary>
  <ul>
    <li>python311Packages.lxmf</li>
    <li>python311Packages.lxmf.dist</li>
    <li>python311Packages.nomadnet</li>
    <li>python311Packages.nomadnet.dist</li>
    <li>python311Packages.rns</li>
    <li>python311Packages.rns.dist</li>
    <li>python312Packages.lxmf</li>
    <li>python312Packages.lxmf.dist</li>
    <li>python312Packages.nomadnet</li>
    <li>python312Packages.nomadnet.dist</li>
    <li>python312Packages.rns</li>
    <li>python312Packages.rns.dist</li>
  </ul>
</details>

Erethon commented:
[rns 0.8.8](https://github.com/markqvist/Reticulum/releases/tag/0.8.8) was released 5 hours ago fyi

Erethon commented on pkgs/development/python-modules/nomadnet/default.nix:
```diff
@@ -12,7 +12,7 @@
 buildPythonPackage rec {
   pname = "nomadnet";
-  version = "0.5.4";
+  version = "0.5.5";
```
```suggestion
  version = "0.5.6";
```

Erethon commented on pkgs/development/python-modules/nomadnet/default.nix:
```diff
@@ -21,7 +21,7 @@ buildPythonPackage rec {
     repo = "NomadNet";
     rev = "refs/tags/${version}";
-    hash = "sha256-4dHxwTHDe8aE/FFtf9jhOO1Wf3uU7KjBa/ngj8o5iMY=";
+    hash = "sha256-uT5W/jea6vQ3XvtaTFcgvUz6K7FB3rEhtZGpjm0w7WY=";
```
```suggestion
    hash = "sha256-dwymJIsMDeVsG7eF45CgUPlZf3sEdwnxZ8OxT+gEQCs=";
```

Erethon commented on pkgs/development/python-modules/rns/default.nix:
```diff
@@ -13,7 +13,7 @@
 buildPythonPackage rec {
   pname = "rns";
-  version = "0.8.6";
+  version = "0.8.7";
```
```suggestion
  version = "0.8.8";
```

Erethon commented on pkgs/development/python-modules/rns/default.nix:
```diff
@@ -22,7 +22,7 @@ buildPythonPackage rec {
     repo = "Reticulum";
     rev = "refs/tags/${version}";
-    hash = "sha256-LvtiK/j6EuXqBOj04x6aWoLOfhukFQxVsEzT/SWvcHU=";
+    hash = "sha256-QDX+pOzn4o3gNL16e/26VYQAROChxAT5eEFcsokg7uc=";
```
```suggestion
    hash = "sha256-Vsh5C0IlOz8/Jw0ya1bOGsNiBQTXJwTWUBcDFs5Zp+0=";
```

Erethon reviewed (commented):
Added suggestions to update to the latest RNS (0.8.8) and nomadnet (0.5.6) versions.

fabaff committed:
python312Packages.rns: 0.8.7 -> 0.8.8

Diff: https://github.com/markqvist/Reticulum/compare/refs/tags/0.8.7...0.8.8

Changelog: https://github.com/markqvist/Reticulum/releases/tag/0.8.8

fabaff committed:
python312Packages.nomadnet: 0.5.5 -> 0.5.6

Diff: https://github.com/markqvist/NomadNet/compare/refs/tags/0.5.5...0.5.6

Changelog: https://github.com/markqvist/NomadNet/releases/tag/0.5.6

fabaff commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 363954`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 12 packages built:</summary>
  <ul>
    <li>python311Packages.lxmf</li>
    <li>python311Packages.lxmf.dist</li>
    <li>python311Packages.nomadnet</li>
    <li>python311Packages.nomadnet.dist</li>
    <li>python311Packages.rns</li>
    <li>python311Packages.rns.dist</li>
    <li>python312Packages.lxmf</li>
    <li>python312Packages.lxmf.dist</li>
    <li>python312Packages.nomadnet</li>
    <li>python312Packages.nomadnet.dist</li>
    <li>python312Packages.rns</li>
    <li>python312Packages.rns.dist</li>
  </ul>
</details>
